### PR TITLE
feat: initial version

### PR DESCRIPTION
**Pull request by raszi**

### CODE REVIEW

File: docker.10s.js [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+/*jshint esversion: 6 */
+
+const _ = require('lodash');
+const bitbar = require('bitbar');
+const async = require('async');
+const Docker = require('dockerode');
+const Handlebars = require('handlebars');
+
+const states = ["running", "created", "restarting", "paused", "exited",	"dead"];
```
Comment (thenonameguy) [inline docker.10s.js]: Extra whitespace.

File: docker.10s.js [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+const darkStateColors = {
+  created: "green",
+	restarting: "yellow",
+	running: "green",
```
Comment (thenonameguy) [inline docker.10s.js]: Borked indentation

File: docker.10s.js [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+  paused: "white",
+  exited: "white",
+  dead: "red"
+};
+
+const lightStateColors = {
+  created: "green",
+	restarting: "yellow",
+	running: "green",
```
Comment (thenonameguy) [inline docker.10s.js]: Same here.

File: docker.10s.js [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+  return _.head(getNames(cnt));
+});
+
+Handlebars.registerHelper('firstTag', firstTag);
+
+Handlebars.registerHelper('autoName', (img) => {
+  var baseName = name = _.head(firstTag(img).split(':', 2));
+  var takenNames = _.flatMap(img.containers, getNames);
+
+  var name;
```
Comment (thenonameguy) [inline docker.10s.js]: `name` is already defined on line 59.

File: docker.10s.js [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+
+  for(var i = 1, name = baseName; _.includes(takenNames, name); i++) {
```
Comment (thenonameguy) [inline docker.10s.js]: This `name = baseName` is already done on line 59.

File: docker.10s.js [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+  return _.chain(input)
+    .mapValues(_.partial(_.map, _, map))
+    .toPairs()
+    .sortBy(sortBy)
+    .flatMap(mapMenuItems)
+    .value();
+};
+
+var parseResults = (err, results) => {
+  // TODO handle errors
```
Comment (thenonameguy) [inline docker.10s.js]: Do you want to do anything about this TODO in this PR?

Comment (raszi) [inline docker.10s.js]: No, I don't actually know what should I do with that :)

**Comment by thenonameguy** [comment]
LGTM

**Pull request merged**